environ/src/environ/core.cljs

```diff
@@ -0,0 +1,10 @@
+(ns environ.core
+  (:require [environ.util :as util]))
+
+(defonce ^{:doc "A map of environment variables."}
+  env
+  (->> js/process.env
```

Comment:
We're not necessarily working off Node, so you need to put in a check to make sure process.env exists and is an object.